### ISSUE TITLE
Is Resource owner flow possible or not and if yes how?

### ISSUE THREAD

**Issue by ohcibi** [jazzband/django-oauth-toolkit#604]
I'm coming from #581 which involves some knowledge and some questions that are kinda off topic there and therefore I made a new issue.

The problem I have is that the documentation is to centric on using some html views that ship with this plugin. My goal is to create a single page application that authorizes itself in a similar way as any other potential client would do via oauth2. For this I basically want to follow the resource owner flow as described in the rfc here: https://tools.ietf.org/html/rfc6749#section-4.3

So I want to 

1. let user enter credentials in a form
2. make a POST request with those credentials
3. receive the oauth token as a response when the credentials are correct

Is this even possible with django-oauth-toolkit or is dot not meant to be used for this flow? Please explain how if the answer is that its possible. I'm happy to PR an understandable documentation if things are clear then.

**Comment by jdelic** [comment]
Just for completeness sake, I *think* that I answered this [here in #581](https://github.com/jazzband/django-oauth-toolkit/issues/581#issuecomment-397439731).

**Comment by ohcibi** [comment]
@jdelic yep. I'll have a look into that and thanks for the detailed response. I just create this issue to separate the discussion from the problem discussed in the other issue.
